db/migrate/20240828192509_create_questions.rb

```diff
@@ -0,0 +1,10 @@
+class CreateQuestions < ActiveRecord::Migration[6.1]
+  def change
+    create_table :questions do |t|
+      t.string :body, null: false
```

Comment:
```suggestion
      t.text :body, null: false
```
предлагаю вместо `string` использовать `text`
по умолчанию `string` ограничено 255 символами. Это стандартное ограничение для `VARCHAR` в большинстве SQL баз данных, в то время как `text` фактически не имеет ограничений по количеству символов и хорошо подходит для хранения длинных текстовых данных. вдруг у нас будет длинное тело вопроса?